CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* Add new `Lint/NonDeterministicRequireOrder` cop. ([@mangara][])
```

Comment:
Could you add a link to this PR? (See other changelog entries for examples.) Each changelog entry usually links to an issue or to a PR if there is no related issue.